src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/GoldPan.java

```diff
@@ -160,16 +181,46 @@ public void updateRandomizer() {
 
     @Override
     public @Nonnull List<ItemStack> getDisplayRecipes() {
-        List<ItemStack> recipes = new LinkedList<>();
+        List<ItemStack> recipes = new ArrayList<>();
 
         for (GoldPanDrop drop : drops) {
-            if (drop.getValue() > 0) {
-                recipes.add(new ItemStack(getInputMaterial()));
+            if (drop.getValue() <= 0) {
+                continue;
+            }
+
+            for (Material material : getInputMaterials()) {
+                recipes.add(new ItemStack(material));
                 recipes.add(drop.getOutput());
             }
         }
 
         return recipes;
     }
 
+    /**
+     * This returns whether the {@link GoldPan} accepts the {@link ItemStack} as an input
+     *
+     * @param itemStack
+     *            The {@link ItemStack} to check
+     *
+     * @return If the {@link ItemStack} is valid
+     */
+    public boolean isValidInput(ItemStack itemStack) {
+        Material material = itemStack.getType();
```

Comment:
This is the issue, if itemStack == null this fails and that's what triggers the error. Same would happen with the nether gold pan